wagtail/images/views/chooser.py

```diff
@@ -72,10 +72,16 @@ def get_creation_form_kwargs(self):
 class BaseImageChooseView(BaseChooseView):
     template_name = "wagtailimages/chooser/chooser.html"
     results_template_name = "wagtailimages/chooser/results.html"
-    per_page = 12
     ordering = "-created_at"
     construct_queryset_hook_name = "construct_image_chooser_queryset"
 
+    @property
+    def per_page(self):
+        # Make per_page into a property so that we can read back WAGTAILIMAGES_CHOOSER_PAGE_SIZE
+        # at runtime. This means we can no longer set it as an attribute on the viewset, but that's
+        # fine.
```

Comment:
This isn't relevant after the switch to use the sentinel value, I think?

```suggestion
        # at runtime.
```